hs_core/management/commands/repair_resource.py

```diff
@@ -25,47 +25,100 @@ class Command(BaseCommand):
     help = "Check synchronization between iRODS and Django."
 
     def add_arguments(self, parser):
-
-        # a list of resource id's, or none to check all resources
         parser.add_argument('resource_ids', nargs='*', type=str)
-
+        parser.add_argument('--days', type=int, dest='days', help='include resources updated in the last X days')
```

Comment:
modified this management command so that we can easily call it from a Jenkins job